config.md

```diff
@@ -101,6 +101,11 @@ For POSIX platforms the `mounts` structure has the following fields:
 * **`type`** (string, OPTIONAL) The type of the filesystem to be mounted.
     * Linux: filesystem types supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660"). For bind mounts (when `options` include either `bind` or `rbind`), the type is a dummy, often "none" (not listed in */proc/filesystems*).
     * Solaris: corresponds to "type" of the fs resource in [zonecfg(1M)][zonecfg.1m].
+* **`uidmappings` (array of type LinuxIDMapping, OPTIONAL). The file system mapping to convert uid and gid from value written in the container's image into desired value. This mapping contains of following fields: ContainerID, HostID, Size. These values would be written into temporary user namespace's /proc/$pid/uid_map and /proc/$pid/gid_map as followes "ContainerID HostID Size". The uid and gid of the image file system would be mapped inside kernel from ContainerdID to HostID. The Size field is a range for mapping, as an example.
```

Comment:
```suggestion
* **`uidMappings`**, **`gidMappings`**  (array of type LinuxIDMapping, OPTIONAL). The file system mapping used to convert UIDs and GIDs from the source file system. This mapping contains the following fields: ContainerID, HostID, Size. These values would be written into temporary user namespace's /proc/$pid/uid_map and /proc/$pid/gid_map as follows "ContainerID HostID Size". The UID and GID of the source file system are mapped inside the kernel from ContainerID to HostID. The Size field is a range for mapping, as an example.
```

Reply:
Would it be written to /proc/$pid/uid_map? I don't know what pid refers to here, but if it is the container pid and the container userns uses a different mapping, I guess that is not true?

Can't we just avoid mentioning those implementation details?

Reply:
probably that is better, let's avoid the implementation details